modules/library/referencing/src/test/java/org/geotools/referencing/proj/proj_generatecodes.py

```diff
@@ -0,0 +1,18 @@
+import csv
```

Comment:
This should go in a test/python directory, along with a README explaining how to use it.

Reply:
Done